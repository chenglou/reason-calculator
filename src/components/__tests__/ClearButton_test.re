open Jest;
open Expect;

let _ =

describe "ClearButton" (fun _ => {
  test "renders" (fun _ => {
    let dispatch = fun _ => ();
    let tree = ClearButton.createElement ::dispatch children::[] ()
      |> ReactShallowRenderer.renderWithRenderer;

    expect tree |> toMatchSnapshot;
  });
});
