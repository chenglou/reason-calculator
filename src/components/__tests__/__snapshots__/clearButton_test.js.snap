// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClearButton renders 1`] = `
<GreenButton
  reasonProps={
    Array [
      1,
      Object {
        "data-css-galczn": "",
      },
      [Function],
    ]
  }
/>
`;
